sound/soc/sof/trace.c

```diff
@@ -30,18 +30,23 @@ static size_t sof_wait_trace_avail(struct snd_sof_dev *sdev,
 				   loff_t pos, size_t buffer_size)
 {
 	wait_queue_entry_t wait;
+	u32 host_offset;
```

Comment:
this is confusing. You are using 'host_offset but this field was removed from the structures so now it appears as a different concept. did you mean trace_offset?

Reply:
host_offset is a local variable here to temporary store the latest buffer offset, if it is confusing to you, how about changing it to buffer_offset?

Reply:
it helps if the local variable has the same name as the field in the structure you compare against.e.g.
````c
trace_offset = (sdev->trace_init_offset + sdev->trace_offset) %
		      buffer_size;
````

otherwise, you might as well call it foobar

Reply:
OK, let me change it to trace_offset.